docs/root/configuration/http/http_filters/_include/aws-request-signing-filter-sigv4a.yaml

```diff
@@ -0,0 +1,53 @@
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 80
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          codec_type: AUTO
+          stat_prefix: ingress_http
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: app
+              domains:
+              - "*"
+              routes:
+              - match:
+                  prefix: "/"
+                route:
+                  cluster: versioned-cluster
+          http_filters:
+          - name: envoy.filters.http.aws_request_signing
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.aws_request_signing.v3.AwsRequestSigning
+              service_name: vpc-lattice-svcs
+              signing_algorithm: aws_sigv4a
```

Comment:
this has to be all caps? `AWS_SIGV4A`

Reply:
in the config file lower case is acceptable. this is validated in the test cases also.

Reply:
have updated - appears to be bugging out on one of the builds without it